scripts/pre_commit_linter.py

```diff
@@ -124,6 +124,12 @@
     'integrations_dev/*', '*.svg', '*.png', '*.zip', '*.ico', '*.jpg',
     '*.min.js', 'assets/scripts/*', 'core/tests/data/*', '*.mp3')
 
+WHITELISTED_DIRECTIVE_IDS = [
```

Comment:
The name is too generic. Perhaps something like: NG_TEMPLATE_WHITELIST or NG_TEMPLATE_DIRECTIVE_IDS_WHITELIST.